### DISCUSSION TITLE
Installation Error: Atri

### DISCUSSION THREAD

**Discussion by Hareesh061** [Atri-Labs/atrilabs-engine#688] [Help: Installation & Start]
When I started to install Atri I got some error. Can anyone help me out ...

ERROR:: --system is intended to be used for pre-existing Pipfile installation, not installation of specific packages. Aborting.

contact mail: hareeshnjscs@gmail.com

**Comment by Shubhodeep100** [comment]
1. `mkdir new_app`
(Step 1 is optional, if you have already made a directory, then skip step 1 and follow from step 2 serially.)
2. `pip uninstall virtualenv`
3. `pip uninstall pipenv`
4.  `pip install pipenv`
5.  `python -m pipenv install atri` (Atri packages and dependencies will start to download)
6.  `python -m pipenv shell`  (A virtual environment will start)
7.   `atri start`   (Atri -Editor will start)

( If you encounter any error message on any step, till your Atri-Editor gets started,  you can reply back here in this discussion with a screenshot of your error-message-screen for better understanding) Hope this helps.

**Reply by darshitac11** [reply]
@Shubhodeep100 - thanks for taking the time and helping others!

**Reply by Shubhodeep100** [reply]
My pleasure mam. Your welcome.

**Comment by Hareesh061** [comment]
Hi there,

As per your instruction I uninstalled virtualenv and pipenv then I
installed again as mentioned in step 4. When I tried to install atri as per
step 5 I'm getting an error.

Error:No module name pipenv

But it's installed already. I have the default python3 version 3.9.6 and
new python version 3.10.9. When I execute the atri installation command
it's redirected to the new version python path with "no module error". If I
try to install pipenv it shows it's already there. So I think maybe these
python versions are playing between them.(default python3 path and new
installed python path are different but I added both of them in the
environment variable system place) If you understand what's going on please
reply to me when you are free(It's not part of the internship, I wanted to
know how its works). I added some pics of the command line.

Thank you...

**Reply by Shubhodeep100** [reply]
I know its creating problems and the process is not getting executed at once. Even I had to restart the process from scratch a several times. The commands which I have written above (for opening Atri-Editor) are a bit different than what its given in the Atri-Lab's Docs. I had to edit, retry with several different procedures to finally & then I came up with the Commands which actually worked in my system, leading to starting of my Atri-Editor. I'll suggest you to start-over the process again.
